lib/katello/repository_types/ostree.rb

```diff
@@ -27,6 +27,7 @@
                        model_name: lambda { |pulp_unit| pulp_unit["name"] },
                        model_version: lambda { |pulp_unit| pulp_unit["checksum"] },
                        uploadable: true,
+                       generic_browser: true,
```

Comment:
if we want to enable the browser on this PR, should the issue/commit also be updated to say something like "also enables generic content browser for ostree refs"?

Reply:
I'm happy to add that to the issue if you think it's important. I thought it it was inherent with this change.

Reply:
Now that I've looked at the CVV changes more, I agree with you